app/dcr_monitor/reconciler_test.go

```diff
@@ -33,7 +33,7 @@ func (f *FakeTEEProvider) GetInstanceStatus(instanceName string) (string, error)
 	return status, nil
 }
 
-func (f *FakeTEEProvider) LaunchInstance(instanceName string, image string, digest string, extraEnvs map[string]string) error {
+func (f *FakeTEEProvider) LaunchInstance(instanceName string, image string, digest string, extraEnvs map[string]string, cpuCount int32, diskSize int32) error {
```

Comment:
Can you please refactor this interface such that it gets `db.Job` instead of a bunch of parameters?

Reply:
done